hls4ml/templates/vivado/build_prj.tcl

```diff
@@ -65,6 +65,7 @@ if {$opt(reset)} {
   open_solution "solution1"
 }
 catch {config_array_partition -maximum_size 4096}
+config_compile -name_max_length 60
```

Comment:
What's this for?

Reply:
It is the length of the function name. It is very useful when looking at reports. You can figure out exactly which layer you are looking at. Instead of `dense_U0` you would get full template info, like `dense_array_array_ap_fixed_16_7_5_3_0_8u_config6_U0`

Reply:
Nice!